resident/resident-service/src/main/java/io/mosip/resident/validator/RequestValidator.java

```diff
@@ -844,6 +846,28 @@ public void validateUpdateRequest(RequestWrapper<ResidentUpdateRequestDto> reque
 		}
 	}
 
+	private void validateLanguageCodeInIdentityJson(JSONObject identity) {
+		// Get a set of entries
+		for (Map.Entry entry : (Iterable<Map.Entry>) identity.entrySet()) {
+			// Retrieve the key and value of each entry
+			String key = (String) entry.getKey();
+			Object value = entry.getValue();
+			if (value instanceof ArrayList){
+				ArrayList valueArray = (ArrayList) value;
+				for(Object valueInList: valueArray){
+					if(valueInList instanceof LinkedHashMap){
+						LinkedHashMap<String, String> valueInListMap = (LinkedHashMap<String, String>) valueInList;
+						if(valueInListMap.containsKey(ResidentConstants.LANGUAGE)){
+							validateLanguageCode(valueInListMap.get(ResidentConstants.LANGUAGE));
+						}
```

Comment:
* we should also check "language" attribute not null and not empty.
* And we should also check "value" attribute not null and not empty.

Reply:
For "language" attribute null check is already taken care of in validateLanguageCode() method.
for the value, I have checked.